output/Html.php

```diff
@@ -146,7 +146,7 @@ protected function build_sorter() {
 	 * @return string Markup for the column sorter controls.
 	 */
 	protected function build_toggler() {
-		$out = '<button class="qm-toggle" data-on="+" data-off="-" aria-pressed="false">+</button>';
+		$out = '<button class="qm-toggle" data-on="+" data-off="-" aria-expanded="false">+<span class="screen-reader-text">Toggle button</span></button>';
```

Comment:
Hey @ivdimova, add a space before the Toggle like `<span class="screen-reader-text"> Toggle button</span>`
That way it's pronounces better.
Then It's ok for me.

The text still isn't very unique, but it's better than it was.

Reply:
Thanks Rian. @ivdimova Can you also add the missing i18n that we just chatted about?